front/pages/api/w/[wId]/assistant/conversations/[cId]/messages/index.ts

```diff
@@ -109,8 +109,8 @@ async function handler(
 
       const { content, context, mentions } = bodyValidation.right;
 
-      // Not awaiting this promise on prupose. We want to answer "OK" to the client ASAP and process
-      // the events in the background.
+      /* postUserMessageWithPubSub returns swiftly since it only waits for the
```

Comment:
Please use `//` comments out of consistency (we generally don't use any /* */ within code block)